ddtrace/contrib/aiobotocore/patch.py

```diff
@@ -21,7 +27,11 @@ def patch():
     setattr(aiobotocore.client, '_datadog_patch', True)
 
     wrapt.wrap_function_wrapper('aiobotocore.client', 'AioBaseClient._make_api_call', _wrapped_api_call)
-    Pin(service='aws', app='aws', app_type='web').onto(aiobotocore.client.AioBaseClient)
+    Pin(
+        service=config.aiobotocore['service_name'],
+        app='aws',
+        app_type='web',
+    ).onto(aiobotocore.client.AioBaseClient)
```

Comment:
So this is more or less the 'happy'/simple path, that's very similar to celery's approach. For comparison